CONTRIBUTING.rst

```diff
@@ -97,10 +100,14 @@ e.g. authorAwardsSparql
 
 4. Add that name to the sparql-to-data table
 
+.. code:: python
+
    sparqlToDataTable(authorAwardsSparql, "#author-awards");
 
 5. Add some table formatting
 
+.. code:: html
+
    <h2 id="AuthorAwards">Author Awards</h2>
```

Comment:
I think `<h2 id="Author-awards">Author awards</h2>` would be better, i.e., lower case section words, except the first. A make a dash in the id for the space.

Reply:
done